pureStorageDriver/README.md

```diff
@@ -116,7 +116,7 @@ The following table lists the configurable parameters and their default values.
 | `database.tolerations`                         | [Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts)                                                            | `[]`                                          |
 | `database.affinity`                            | [Affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)                                                  | `{}`                                          |
 | `images.plugin.name`                           | The image name to pull from                                                                                                                                | `purestorage/k8s`                             |
-| `images.plugin.tag`                            | The image tag to pull                                                                                                                                      | `6.0.0-rc2`                                   |
+| `images.plugin.tag`                            | The image tag to pull                                                                                                                                      | `6.0.0-rc3`                                   |
```

Comment:
Just realized that cockroach-operator and db-deployer tags aren't listed here, which seems not quite right. Doesn't need to be fixed in this PR, just happened to notice it.

Reply:
@Pure-AdamuKaapan I have added these as part of #51